tests/executors/test_kubernetes_executor.py

```diff
@@ -718,20 +718,42 @@ def test_adopt_launched_task(self, mock_kube_client):
         pod_ids = {ti_key: {}}
 
         executor.adopt_launched_task(mock_kube_client, pod=pod, pod_ids=pod_ids)
-        assert mock_kube_client.patch_namespaced_pod.call_args[1] == {
-            "body": {
-                "metadata": {
-                    "labels": {"airflow-worker": "modified"},
-                    "annotations": annotations,
-                    "name": "foo",
-                }
-            },
-            "name": "foo",
-            "namespace": None,
-        }
+        mock_kube_client.patch_namespaced_pod.assert_called_once_with(
+            body={"metadata": {"labels": {"airflow-worker": "modified"}}},
+            name="foo",
+            namespace=None,
+        )
         assert pod_ids == {}
         assert executor.running == {ti_key}
 
+    @mock.patch("airflow.executors.kubernetes_executor.get_kube_client")
+    def test_adopt_completed_pods(self, mock_kube_client):
```

Comment:
Could add docstring saying what this is testing

Reply:
I am rather opting for veruy long and very descriptive test method names:

`test_adopted_pod_only_update_label_and_nothing_else` 

(I am not joking).

Reply:
The problem is we'd end up with `test_adopt_completed_pods_only_those_from_other_schedulers_and_only_patch_label`. The one for `adopt_launched_task` would be even worse. Let me come up with something.